packages/js/product-editor/src/components/header/publish-button/publish-button.tsx

```diff
@@ -36,9 +36,9 @@ export function PublishButton(
 		onPublishSuccess( savedProduct: Product ) {
 			recordProductEvent( 'product_update', savedProduct );
 
-			const noticeContent = isCreating
-				? __( 'Product successfully created.', 'woocommerce' )
-				: __( 'Product published.', 'woocommerce' );
+			const noticeContent = isPublished
```

Comment:
Noted this in the other PR, but won't let it block this PR since it's fixed there.

https://github.com/woocommerce/woocommerce/pull/38507#discussion_r1212336166